README.md

```diff
@@ -86,6 +86,20 @@ $databases['default']['default'] = array (
 );
 ```
 
+### Memcached settings
+
+To enable memcached add these settings to your `settings.php` file.
+Replace `[PATH TO MEMCACHE MODULE]` with path to memcached module in your project.
+
+```php
+// Memcache
+$conf['cache_backends'][] = '[PATH TO MEMCACHE MODULE]/memcache/memcache.inc';
+$conf['cache_default_class'] = 'MemCacheDrupal';
+$conf['memcache_servers'] = array(
```

Comment:
cache_form should be excluded from memcache, since memcache is considered unreliable.

Here's what Acquia recommends (last line):

```
$conf['cache_backends'][] = './sites/all/modules/memcache/memcache.inc';
$conf['cache_default_class'] = 'MemCacheDrupal';
$conf['cache_class_cache_form'] = 'DrupalDatabaseCache';
```